### DISCUSSION TITLE
Adding multiple callsigns by import?

### DISCUSSION THREAD

**Discussion by Michael-AUG** [magicbug/Cloudlog#2192] [General]
Hi Peter
I've just become the Callsign Manager for SKCC UK and need to add about eight different callsigns. They're basically just the UK callsign variations on a same call. Is there a way I can import these in the back end (perhaps as some sort of CSV or the like)?
No huge problem if not, but if I can be economical it would save a bit of time.
Thanks
73 Michael GM5AUG

**Comment by AlbrechtDIETRICH** [comment]
Hi Michael,
what do you mean with "They are basically just the UK callsign variations on a same call."?
Is it the same call always with different operators or different call signs (e. g. /SOTA, /m, /p, etc.)?
There a several way to import but depending on your needs (e. g. different LogBooks, different locations / operators).
Please advice and I will try to help you!
vy 73 de DC5GD, Al
